fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4

```diff
@@ -433,13 +433,13 @@ DECIMAL_VALUE
     ;
 
 FLOAT_LITERAL
-    : DIGIT+ EXPONENT? 'F'
-    | DECIMAL_DIGITS EXPONENT? 'F' {isValidDecimal()}?
+    : DIGIT+ EXPONENT? 'F'?
```

Comment:
we should not change this, but to add another `DECIMAL_VALUE` with no suffix. And like Integral type, decide what real type to use in LogicalPlanBuilder

Reply:
Is there a way to remove the suffix support  from FLOAT/DOUBLE's lex, since Doris doesn't have such lex